ValidateWorkspace target in 1.2.0 fails because task is not found

## Summary
The released Build SDK 1.2.0 exposes a `ValidateWorkspace` MSBuild target, but invoking it directly fails because the `ValidateWorkspace` task cannot be found.

## Repro
1. Reference released `TALXIS.DevKit.Build.Sdk` 1.2.0.
2. Run:
   `dotnet msbuild src/Solutions.DataModel/Solutions.DataModel.csproj -target:ValidateWorkspace -v:m`

## Actual
MSBuild prints:
`error MSB4036: The "ValidateWorkspace" task was not found.`

## Expected
The target should be callable directly and load its task successfully.

## Additional context
A normal `dotnet build` still succeeds, and the logs show the legacy validation steps (`Duplicate GUID check`, `Quick Find view validation`) rather than a clearly running unified validator. That suggests the release may not actually be wired to run the new validator end-to-end.

--- a/src/Dataverse/Tasks/msbuild/tasks/TALXIS.DevKit.Build.Dataverse.Tasks.targets
+++ b/src/Dataverse/Tasks/msbuild/tasks/TALXIS.DevKit.Build.Dataverse.Tasks.targets
@@ -39,4 +39,5 @@
     <UsingTask TaskName="ValidateQuickFindViews" AssemblyFile="$(TasksAssembly)" />
     <UsingTask TaskName="EnsureAllCustomizationsNodes" AssemblyFile="$(TasksAssembly)" />
     <UsingTask TaskName="ValidatePcfDependencies" AssemblyFile="$(TasksAssembly)" />
+    <UsingTask TaskName="ValidateWorkspace" AssemblyFile="$(TasksAssembly)" />
 </Project>

